Update oss-sustainability-challenges.md (#3689)
the-turing-way/the-turing-way: book/website/collaboration/oss-sustainability/oss-sustainability-examples.md

# Examples of Open Source Business Models

> Open source has been an important theme for AI communities in academia and industry. Popular open source software libraries like OpenCV and Scikit-Learn and open dataset and model providers like Hugging Face and Open AI have shaped how AI algorithms are discovered, improved, shared, and deployed. While each organisation pursued different paths towards sustainability, reflective of when they were founded, some themes emerge around important initial steps.

Reviewer: ```suggestion
Open source has been an important theme for AI communities in academia and industry. 
Popular open source software libraries like [OpenCV](https://opencv.org/) and [Scikit-Learn](https://scikit-learn.org/), and open dataset and model providers like [Hugging Face](https://huggingface.co/) and [OpenAI](https://openai.com/) have shaped how AI algorithms are discovered, improved, shared, and deployed. 
While each organisation pursued different paths towards sustainability, reflective of when they were founded, some themes emerge around important initial steps.
```

> Of the four organisations of focus in this case study, OpenCV and Scikit-Learn started as community-driven projects. Both had a small group of motiviated individuals with a mission of creating "optimised" and "accesible" code within their domains and found other interested supporters within their organisation or networks to help continue building out the project. These grassroots efforts served to generate interest among many across academia and industry and foster a sense of community among these different user groups. The Scikit-Learn community has recently spun out a company called [Probabl](https://techcrunch.com/2024/02/01/probabl-is-a-new-ai-company-built-around-popular-library-scikit-learn/?guccounter=1&guce_referrer=aHR0cHM6Ly93d3cuZ29vZ2xlLmNvbS8&guce_referrer_sig=AQAAAFCXO8XIZzp8FeD8sLQNQ70Zxza_SBRbDTbwcmZLpIqLQMuDvh7_fXvKsEAnVwaJ0P9RMvta-MYuE_Qi7-TJKoJ97Ab-nyHJHYncFVWTAdrdFc4Ao4qOTx6Y4PD73yJIi3-gFt6w2S3IVYcA4PnbNAg5xVbdrtXPVF-CsnFvhX9T), to offer services such as training and consulting relating to the scikit-learn codebase. 

Reviewer: ```suggestion
## Community-led Sustainability Model
Of the four organisations of focus in this case study, OpenCV and Scikit-Learn started as community-driven projects. 
Both had a small group of motiviated individuals with a mission of creating "optimised" and "accesible" code within their domains and found other interested supporters within their organisation or networks to help continue building out the project. 
These grassroots efforts served to generate interest among many across academia and industry and foster a sense of community among these different user groups. 
The Scikit-Learn community has recently spun out a company called [Probabl](https://techcrunch.com/2024/02/01/probabl-is-a-new-ai-company-built-around-popular-library-scikit-learn/), to offer services such as training and consulting relating to the scikit-learn codebase. 
```

Reviewer: Got rid of all the tracking URL details

> In contrast, OpenAI and Hugging Face started as investor-funded organisations to tackle specific topics such as AI Safety and Chatbots. While Open AI began as a self-defined open source non-for-profit and Hugging Face as an AI product company, both have taken different paths as they have grown. After a private investment from Microsoft, Open AI split into a not-for-profit and for-profit arm. Around the same time, they also began exploring new models of releasing code and models in a more closed or tiered-access way, different from the original "open appraoch." In contrast, Hugging Face has moved in a different direction, transitioning from an AI product buisness model to become an open source AI model and data platform, with over 100,000 pre-trained models and 10,000 datasets hosted on the platform. It has also built [BigScience](https://bigscience.huggingface.co/), a multi-language Large Language Model (LLM) openly with collaborators around the world.

Reviewer: ```suggestion
## Investor-led Sustainability Model
In contrast, OpenAI and Hugging Face started as investor-funded organisations to tackle specific topics such as AI Safety and Chatbots. 
While Open AI began as a self-defined open source non-for-profit and Hugging Face as an AI product company, both have taken different paths as they have grown. 
After a private investment from Microsoft, Open AI split into a not-for-profit and for-profit arm. 
Around the same time, they also began exploring new models of releasing code and models in a more closed or tiered-access way, different from the original "open appraoch." 
In contrast, Hugging Face has moved in a different direction, transitioning from an AI product buisness model to become an open source AI model and data platform, with over 100,000 pre-trained models and 10,000 datasets hosted on the platform. 
It has also built [BigScience](https://bigscience.huggingface.co/), a multi-language Large Language Model (LLM) openly with collaborators around the world.
```
